### DISCUSSION TITLE
Browser Limit Width in Destkop Mode

### DISCUSSION THREAD

**Discussion by Gatos90** [AndreyPavlenko/Fermata#378] [Q&A]
How can I restrict the width of the browser when it's in desktop mode? The issue I'm facing is that the browser extends beyond the width allocated for the app, making it extremely difficult to scroll to the right while a video is playing.

**Comment by AndreyPavlenko** [comment]
Most probably, the problem is not in the browser width, but in the content width, that exceeds the browser. Have you tried to open the same page in the desktop mode in the Chrome mobile browser?

**Reply by Gatos90** [reply]
I've just checked, and when I use Primevideo on my Android smartphone, it doesn't stretch and is bigger the entire screen width. I've considered two potential fixes:
1. Implementing arrows for horizontal scrolling.
2. Adjusting the website's width by injecting JavaScript.

I believe the first option is less prone to errors.

**Reply by AndreyPavlenko** [reply]
Another option - use the screen mirror and the native Prime Video app.
